apps/galaxy/src/services/orders/useOrders.ts

```diff
@@ -52,7 +65,44 @@ const useGetOrders = () => {
     getOpenOrders();
   }, []);
 
-  return { lists };
-};
+  const handleAcceptOrderSubmit = async (data: OrderIdList) => {
+    setIsLoading(true);
+    const token = getToken() || "";
+    try {
+      await axios.post(`/franchisee/${getFranchiseeId(token)}/accept_orders`, {
+        orders: data,
+      });
+      toast({
+        title: "Accept Successfully",
+        status: "info",
+        duration: 6000,
+        position: "top",
+        isClosable: true,
+      });
+      getOpenOrders();
+    } catch (error) {
+      const err = error as AxiosError;
```

Comment:
So based on one of the previous conversation made by @kritomomo, it's better if we do

```ts
if (error instanceof AxiosError) {
  // do something
}
```